src/content/guides/farmlands-card-partner-support.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Farmlands Card Partner Support Guide
+img: /farmlands-pos-integration-cover.jpg
+description: How to for Farmlands Card Partners and Card Holders to request support from the relevant company and teams to get the help they need.
+nav:
+  path: Connections/Farmlands
+  title: Support Guide
+  order: 2
+---
+
+If you are a Farmlands Card Partner needing extra support this is the guide for you. Here you will find who you need to reach out to and how you can get in touch with them.
+
+## Support for Portal Users
+
+1. If you require assistance with Portal functionality, contact Centrapay at support@centrapay.com or Farmlands at card.specialist@farmlands.co.nz
```

Comment:
Do these email links render as `mailto:` links?

`mailto:` links are a nice UX touch

Reply:
I will add those in now

Reply:
Good news! Astro automatically recognises that those are email addresses and opens the mail client when clicked. Therefore the mailto tag is unnecessary.

Reply:
That's cool 😎